### PR TITLE
Add menu options to add element files: entity, relationship, diagram

### PR DESCRIPTION
**Pull request by martin-fleck-at**
- Add entries to 'File' main menu
- Add entries to explorer context menu
- Add entries to explorer toolbar

- Re-use file name as element ID

newElements

### CODE REVIEW

File: packages/core/src/browser/new-element-contribution.ts [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+const NEW_ELEMENT_NAV_MENU = [...NavigatorContextMenu.NAVIGATION, '0_new'];
+const NEW_ELEMENT_MAIN_MENU = [...CommonMenus.FILE, '0_new'];
+
+interface NewElementTemplate extends Command {
+    label: string;
+    fileExtension: string;
+    content: string | ((name: string) => string);
+}
+
+const NEW_ELEMENT_TEMPLATES: NewElementTemplate[] = [
```
Comment (harmen-xb) [inline packages/core/src/browser/new-element-contribution.ts:22]: @martin-fleck-at 

I see you add Entity, Relationship and Diagram in this order to this collection, but it doesn't appear in this order everywhere.

For the 'folder'-menu it is correct, but for the context and file menu it's not adhering the order of this array. Can you adjust it so the order is adhered?

File: packages/core/src/browser/new-element-contribution.ts [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+    protected removeFileExtension(name: string, template: NewElementTemplate): string {
+        return name.endsWith(template.fileExtension) ? name.slice(0, -template.fileExtension.length) : name;
+    }
+}
+
+@injectable()
+export class CrossModelFileNavigatorContribution extends FileNavigatorContribution {
+    override registerCommands(registry: CommandRegistry): void {
+        super.registerCommands(registry);
```
Comment (harmen-xb) [inline packages/core/src/browser/new-element-contribution.ts:127]: @martin-fleck-at 
If I understand this correctly this also registers the command for the command pallete?

When I type 'Entity' or 'Relationship' in the command pallete I now see the entries, which seems a bit odd. Also I think the entry should be 'New Entity' for clarity for the user.

File: packages/core/src/browser/new-element-contribution.ts (hunk 1 above)
Comment (martin-fleck-at) [reply packages/core/src/browser/new-element-contribution.ts:22]: Yes, I fixed the order in which the items are specified. Good catch, thank you!

File: packages/core/src/browser/new-element-contribution.ts (hunk 2 above)
Comment (martin-fleck-at) [reply packages/core/src/browser/new-element-contribution.ts:135]: Good point! I removed the items from the command palette (using an empty label) for now. I think that is a bit cleaner. If you want, however, we can also add them with the 'New X' label.

Comment (harmen-xb) [reply packages/core/src/browser/new-element-contribution.ts:135]: @martin-fleck-at Please add the option in the command palette:
- New Entity
- New Relationship
- New ...

Comment (martin-fleck-at) [reply packages/core/src/browser/new-element-contribution.ts:135]: Ok, I'll push an update as soon as https://github.com/CrossBreezeNL/crossmodel/pull/28 is merged as I have to update the PR then anyway for the initial file content.

**Pull request merged**